src/avfoundation.cs

```diff
@@ -1836,6 +1838,14 @@ interface AVAudioSessionSecondaryAudioHintEventArgs {
 
 	delegate void AVPermissionGranted (bool granted);
 
+	[Native]
+	public enum AVAudioSessionInterruptionReason : ulong
```

Comment:
missing availability attributes 
we need them even if the headers did not include them
default to the current SDK version `[iOS (14,5][Mac (11,0)][TV (14,5)][Watch (7,4)[MacCatalyst (14,5)]`
since it's an enum you should check which API(s) are using the type - and adjust the platforms to match